packages/app/src/hooks/common/useImagePicker/usePickImage.ts

```
@@ -0,0 +1,53 @@
+
+const usePickImage = ({ selectedImagesUri }: UseImagePicker) => {
+  const pickImage = async (pickerType: PickerType) => {
+    if (pickerType === "camera") {
+      const cameraPermission =
+        await ImagePicker.requestCameraPermissionsAsync();
+
+      if (cameraPermission.status !== "granted") {
+        return;
+      }
+
+      let result = await ImagePicker.launchCameraAsync({
```

ESLint

[error] 'result' is never reassigned. Use 'const' instead.

(prefer-const)